.gitignore

```diff
@@ -1 +1,47 @@
+build/
+bin/
+lib/
+msg_gen/
+srv_gen/
+msg/*Action.msg
+msg/*ActionFeedback.msg
+msg/*ActionGoal.msg
+msg/*ActionResult.msg
+msg/*Feedback.msg
+msg/*Goal.msg
+msg/*Result.msg
+msg/_*.py
```

Comment:
all this msg and srv stuff is a leftover from rosbuild, isn't it? So shouldn't this be removed?